Sources/FluentUI_macOS/Components/MultilinePillPicker/MultilinePillPickerView.swift

```diff
@@ -30,25 +33,18 @@ public final class MultilinePillPickerView: ControlHostingView {
 		fatalError("init(rootView:) has not been implemented")
 	}
 
-	@MainActor public var isEnabled: Bool = true {
-		didSet {
-			updatePicker()
-		}
-	}
-	@MainActor public var labels: [String] {
-		didSet {
-			updatePicker()
-		}
-	}
-	@MainActor public var action: (@MainActor (Int) -> Void)? {
-		didSet {
-			updatePicker()
-		}
-	}
+	@MainActor @Published public var isEnabled: Bool = true
+	@MainActor @Published public var labels: [String]
+	@MainActor @Published public var action: (@MainActor (Int) -> Void)?
+}
+
+/// Private wrapper `View` to map from view model to `MultilinePillPicker`.
+fileprivate struct MultilinePillPickerWrapper: View {
+	@ObservedObject var viewModel: MultilinePillPickerView
 
-	private func updatePicker() {
-		let picker = MultilinePillPicker(labels: labels, action: action)
-			.disabled(!isEnabled)
-		rootView = AnyView(picker)
+	var body: some View {
+		MultilinePillPicker(labels: viewModel.labels,
+							action: viewModel.action)
```

Comment:
Intended whitespace? 

<img width="574" alt="Screenshot 2025-01-28 at 3 49 20 PM" src="https://github.com/user-attachments/assets/55a5ed4c-f1be-43bb-80fc-7221fe980062" />

Reply:
Ah it's GitHub not playing nicely with our existing indentation in FluentUI macOS. Looks right when tabs are set to four spaces instead of eight.